app/components/ProtectedRoute/index.js

```diff
@@ -16,10 +16,7 @@ const ProtectedRoute = ({ isUserVerified, render: C, isLoggedIn, handleLogout, .
       .map((key) => routeConstants[key].route)
       .includes(rest.path) && rest.exact;
 
-  const guardedRedirection = (to, renderProps) => {
```

Comment:
lets remove guardedRedircetion and just use `<Redirect to={to} />` instead

Reply:
done